audit/gather/gather.go

```diff
@@ -496,12 +502,12 @@ func (g *gather) captureServerProfiles(serverInfoMap map[string]*server.ServerIn
 
 			responses, err := g.doReq(context.TODO(), payload, subject, 1)
 			if err != nil {
-				g.log.Errorf("Failed to request %s profile from server %s: %s", profileName, serverName, err)
+				g.log.Errorf("Failed to request %s profile from server %s: %s", profile, serverName, err)
```

Comment:
Could we change these log lines to something like `g.log.Errorf("Failed to request %s (%d) profile from server %s: %s", profile.name, profile.debug, serverName, err)`?

Reply:
Done.